classtime.js

```diff
@@ -214,17 +325,25 @@ function updateVariables() {
 }
 
 function updateText() {
-    document.getElementById('timeToEndOfClass').innerHTML =  getTimeToEndOfCurrentClassString()
-    document.getElementById("nextClass").innerHTML = getClassName(currentClassPeriodIndex+1)
-    document.getElementById("currentClass").innerHTML = getClassName(currentClassPeriodIndex)
-    //document.getElementById('sentence').innerHTML = getSummaryString()
     document.getElementById('time').innerHTML = getCurrentTimeString();
     document.getElementById('date').innerHTML = getCurrentDateString();
+    
+    if (currentClassPeriodIndex == -1) {
+        document.getElementById("schedule").innerHTML = "and there's <strong>no class</strong> today!"
+        $('.schoolDays').css('display', 'none')
+        return
+    }
+
+    $('.schoolDays').css('display', 'inline')
```

Comment:
why is jQuery needed for this? iirc you can just use something like this (probably not totally valid but) `document.getGetElementByClassName("schoolDays").style.display = "inline";`

Reply:
implemented in dev branch